cityehr-quick-start-guide/src/main/xslt/create-topic-html.xslt

```diff
@@ -25,15 +34,29 @@
         </xsl:call-template>
       </head>
       <body>
-        <div id="top-nav">
+        <nav id="top-nav">
           <xsl:apply-templates select="." mode="top-nav"/>
-        </div>
+        </nav>
+
+        <!-- Petal Edit Button -->
+        <section>
+          <xsl:variable name="petal-source-file-uri" select="com:document-uri(.)" />
+          <xsl:variable name="tokens" select="tokenize($petal-source-file-uri, '/')"/>
+          <xsl:variable name="orig-file-name" select="$tokens[last()]"/>
+          <xsl:variable name="petal-source-file" select="substring-after($petal-source-file-uri, $petal-github-repo-name || '/')" />
+          <xsl:variable name="petal-file-name" select="hcom:dita-filename-to-html($orig-file-name)" />
+          <xsl:variable name="petal-full-url" select="concat($petal-base-url, '?ghrepo=', $petal-github-base-url, '/', $petal-github-org-name, '/', $petal-github-repo-name, '&amp;source=', $petal-source-file, '&amp;branch=', $petal-github-branch, '&amp;referer=', $petal-referrer-url, $petal-file-name)" />
```

Comment:
I am not sure that we have the value for `ghrepo` correct? At the moment it will produce:

```
?ghrepo=https://github.com/evolvedbinary/cityehr-documentation
```

The `https://github.com/` part seem superfluous as this is fixed and won't change. I think we should just be able to send:
```
?ghrepo=evolvedbinary/cityehr-documentation
```

We could then remove the `petal-github-base-url` from the XSLT, Maven, and CI scripts.

Reply:
@adamretter Fixed with f8a9252